ORK1Kit/ORK1Kit/Common/ORK1Skin.swift

```diff
@@ -0,0 +1,661 @@
+//
+//  ORKSkin.swift
+//  ORK1Kit
+//
+//  Created by Eric Schramm on 8/31/22.
+//  Copyright © 2022 researchkit.org. All rights reserved.
+//
+
+import UIKit
+
+
+struct ORK1SizeType {
+    let size: CGSize
+    let isiPad: Bool
+    let metrics : [ORK1ScreenMetricSwift : CGFloat]
+    func metric(for metric: ORK1ScreenMetricSwift) -> CGFloat {
+        return metrics[metric] ?? 9999
+    }
+    
+    static let matrixCSV =  """
```

Comment:
For easier editing, could add this to Xcode as an actual .csv file, then use `Bundle(for: self).url(forResource:...)` to load it at runtime.

Reply:
Good idea, I've now embedded a simple CSV as suggested.